.script/tests/idChangeValidatorTest/idChangeValidatorTest.yaml

```diff
@@ -0,0 +1,4 @@
+# In order to test the idCahngeValidator do the following steps:
+# 1. Change the id of some file in the Detections directory.
```

Comment:
nit: maybe add step 4 for unso the changes, although it is clear because the tet filed.